jobs:
    build:
        container: quay.io/skopeo/stable:latest
        continue-on-error: true
        runs-on: ubuntu-latest
        steps:
        -   name: checkout
            uses: actions/checkout@v2
        -   name: login
            run: for i in {1..5}; do skopeo login -u ${{ secrets.REGISTERY_USERNAME }} registry.cn-hangzhou.aliyuncs.com -p ${{ secrets.REGISTERY_PASSWORD }} || true; done
        -   name: perpare
            run: dnf install pip3 -y && pip3 install redis pyyaml tencentcloud-sdk-python-tcr platform
        -   name: sync
            run: REDIS_HOST=${{ secrets.REDIS_HOST }} REDIS_PASSWORD=${{ secrets.REDIS_PASSWORD }} SECRET_ID=${{ secrets.SECRET_ID }} SECRET_KEY=${{ secrets.SECRET_KEY }} python3 sync.py ${{ matrix.syncs }} -v
        strategy:
            matrix:
                syncs:
           
            max-parallel: 10
name: sync
'on':
    pull_request:
        branches:
        - main
    push:
        branches:
        - main
    schedule:
    -   cron: 0 */6 * * *
